cpp/oneapi/dal/algo/jaccard/vertex_similarity_types.cpp

```diff
@@ -15,48 +15,53 @@
  *******************************************************************************/
 
 #include "oneapi/dal/algo/jaccard/vertex_similarity_types.hpp"
-#include <iostream>
 #include "oneapi/dal/algo/jaccard/common.hpp"
 
 namespace oneapi::dal::preview {
 namespace jaccard {
```

Comment:
just curious - what we choose 'jaccard' name? did we consider other options to name the namespace?

Reply:
@andreynv 
other options: jaccard_similarity, jaccard_coeffs, jaccard_coefficients. 
1. similarity --  we have in entity for algorithm domain
2. coeffs does not contain additional information.

Reply:
that's fine but it is important element of overall scheme and requires design and analysis of the options before the code is created. One of the places when different parts can be done differently in the whole lib